src/librustdoc/html/render/mod.rs

```diff
@@ -1928,18 +1928,23 @@ fn document_full(w: &mut Buffer, item: &clean::Item, cx: &Context, prefix: &str,
     }
 }
 
-fn document_stability(
+/// Add extra information about an item such as:
+///
+/// * Stability
+/// * Deprecated
+/// * Required features (through the `doc_cfg` feature)
+fn document_item_info(
     w: &mut Buffer,
     cx: &Context,
     item: &clean::Item,
     is_hidden: bool,
     parent: Option<&clean::Item>,
 ) {
-    let stabilities = short_stability(item, cx, parent);
-    if !stabilities.is_empty() {
-        write!(w, "<div class=\"stability{}\">", if is_hidden { " hidden" } else { "" });
-        for stability in stabilities {
-            write!(w, "{}", stability);
+    let item_infos = short_item_info(item, cx, parent);
```

Comment:
"information" is an [uncountable noun](https://dictionary.cambridge.org/us/grammar/british-grammar/nouns-countable-and-uncountable).
```suggestion
    let item_info = short_item_info(item, cx, parent);
```

Reply:
Absolutely but in this case, I wanted to underline the fact that there were more than one. So even if not valid english, it conveys the information I wanted for users to understand. Sometimes, english is a bit strange. :)

Reply:
Hmm, I think the fact there's multiple is pretty clear from the loop soon after. I don't feel strongly about it though, feel free to keep the name.

Reply:
Then I'll keep the non-english native way. ;)